plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -211,7 +209,10 @@ public static AWSGlueAsync createAsyncGlueClient(GlueHiveMetastoreConfig config,
                 .withMetricsCollector(metricsCollector)
                 .withClientConfiguration(clientConfig);
 
-        requestHandler.ifPresent(asyncGlueClientBuilder::setRequestHandlers);
+        ImmutableList.Builder<RequestHandler2> requestHandlers = ImmutableList.builder();
+        requestHandler.ifPresent(requestHandlers::add);
+        config.getCatalogId().ifPresent(GlueCatalogIdRequestHandler::new);
```

Comment:
I may be missing something but does this line actually does anything? 
I see that new instance of `GlueCatalogIdRequestHandler` will be created but it will be immediately lost because it is not assigned anywhere. Do we want to create it just for a side effect or is it a mistake ?

Reply:
That's my mistake. I was cleaning code just before sending PR and missed to add it to request handlers. Fixed now.